### ISSUE TITLE
Problem with train() function

### ISSUE THREAD

**Issue by taizun-jj202** [taizun-jj202/Neural_Networks#4]
Might have to change how the train function works because my custom dataset class returns (batch_size, 784) tuple which i have to reshape while testing accuracy.

**Comment by taizun-jj202** [comment]
## Incorrect Input to CNN Algo

Input for neural network is fed a 28*28 image, but should have been fed a 1*784 pixel image. Correcting the neural network now, after calculating Convolution layer  parameters.

**Comment by taizun-jj202** [comment]
Error not in train() but in indexing in CNN_MODEL.py file
